### ISSUE TITLE
ESP8266 cloud.arest.io/[deviceid]/id does not return variables

### ISSUE THREAD

**Issue by saliehendricks** [marcoschwartz/aREST#85]
Hi Marco

Testing the cloud.arest api. So glad you decided to add this in...its works so well. I am finding that the /id or / path does not return the defined variables. I have it working perfectly with the ESP on my wifi.

Only this is returned:

{
id: "xxx",
name: "xxx",
connected: true
}

I have created a fork and will try to debug.

Thanks

**Comment by marcoschwartz** [comment]
Hello, thanks! For the /id path it's supposed to return what it currently returns :) However for the / path you are right there is currently a bug, probably linked to the MQTT packet size. If you find a solution let me know!

**Comment by saliehendricks** [comment]
Hi Marco,

In short you were correct. Some detail is left below for anyone else running into issues.

I set debug to 1 and can see that the / command does indeed yield the correct response:

arest.io/[deviceid]/ reaches the ESP and it responds correctly with the variables. The MQTT message is published. My variables are about 352b of JSON. After a few seconds of waiting for the arest.io http request the broker times out. 

In aRest.h, I set the MQTT_MAX_PACKET_SIZE to 512 and 1024 and even 1536 but it still timed-out. Then I tried setting it directly in PubSubClient.h and then I got a response. 

I am using Visual Studio with VS Micro plugin, so perhaps my environment is causing the directives not to take affect. I'll test it in Arduino IDE when I have it open sometime.

Thanks

**Comment by marcoschwartz** [comment]
Indeed, I had a very similar experience with the packet size. I opened an issue at the MQTT library creator so he implements a function to change that from outside of his library :)
